Meta/label-pull-requests.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2022, Luke Wilde <lukew@serenityos.org>
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+module.exports = ({ github, context }) => {
+    const Label = {
+        CommunityApproved: "✅ pr-community-approved",
+        HasConflicts: "⚠️ pr-has-conflicts",
+        IsBlocked: "⛔️ pr-is-blocked",
+        MaintainerApprovedButAwaitingCi: "✅ pr-maintainer-approved-but-awaiting-ci",
+        NeedsReview: "👀 pr-needs-review",
+        Unclear: "🤔 pr-unclear",
+        WaitingForAuthor: "⏳ pr-waiting-for-author",
+    };
```

Comment:
This doesn't need to be defined as a function-local variable, neither most functions below. Let's get rid of some nesting.